Upstream: yes
Reason: ftbts "the taglist extension is not safe for parallel reading"
Url: https://jira.mongodb.org/browse/CDRIVER-3594
From 214c433997ffbed6f36e2ba46218c5beb30a6107 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Roberto=20C=2E=20S=C3=A1nchez?= <roberto@connexer.com>
Date: Mon, 30 Mar 2020 19:37:03 -0400
Subject: [PATCH] CDRIVER-3594 build Sphinx docs without parallelism

---
 build/cmake/SphinxBuild.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/cmake/SphinxBuild.cmake b/build/cmake/SphinxBuild.cmake
index a7f49d79b..f90308486 100644
--- a/build/cmake/SphinxBuild.cmake
+++ b/build/cmake/SphinxBuild.cmake
@@ -40,7 +40,7 @@ function (sphinx_build_html target_name doc_dir)
       ${CMAKE_COMMAND} -E env
       "PYTHONDONTWRITEBYTECODE=1"
       ${SPHINX_EXECUTABLE}
-         -j ${NPROCS} -qEW -b html
+         -qEW -b html
          -c "${CMAKE_CURRENT_SOURCE_DIR}"
          "${CMAKE_CURRENT_SOURCE_DIR}"
          "${SPHINX_HTML_DIR}"
@@ -133,7 +133,7 @@ function (sphinx_build_man target_name)
       ${CMAKE_COMMAND} -E env
       "PYTHONDONTWRITEBYTECODE=1"
       ${SPHINX_EXECUTABLE}
-         -j ${NPROCS} -qEW -b man
+         -qEW -b man
          -c "${CMAKE_CURRENT_SOURCE_DIR}"
          "${CMAKE_CURRENT_SOURCE_DIR}"
          "${SPHINX_MAN_DIR}"
